elegant/clean_timepoint_data.py

```diff
@@ -104,6 +104,15 @@ def remove_timepoint_from_experiment(experiment_root, timepoint, dry_run=False):
 def remove_excluded_positions(experiment_root,dry_run=False):
     """Deletes excluded positions from an experiment directory
 
+    This function deletes position folders from the specified experiment directory,
+    but saves position_metadata and annotations into the 'excluded_positions' subfolder.
+    'excluded_positions' has a parallel structure to the standard experiment root
+    Each position has a folder containing a copy of its position_metadata;
+    an 'annotations' subfolder contains all of the annotations for excluded positions;
+    and, a copy of the old experiment_metadata lies in this subfolder
+    (though subsequent calls to remove_excluded_positions will overwrite this copy of the experiment metadata).
+    Thus, it is recommended that this function be run on completed experiments to prevent data loss.
```

Comment:
Well, not quite. Running the function twice after excluding more positions will create data loss, even if the experiment was completed prior to either run. But see below for a better idea.